backend/src/main/java/club/devcord/devmarkt/entities/auth/Role.java

```diff
@@ -27,4 +28,15 @@ public record Role(
     String name
 ) {
 
+    @JsonValue
+    @Override
+    public String name() {
+        return name;
+    }
+
+    // dirty fix of jackson/graphql serialization error
```

Comment:
Can you explain what would be broken here without?

Reply:
Jackson/GraphQL serializes this whole Entity as a single field, so it only calls the toString() Methode here. Annotations like @Value or similar doesn't work either